zappa/cli.py

```diff
@@ -462,26 +461,32 @@ def unschedule(self):
 
         """
 
-        if self.stage_config.get('events', None):
-            events = self.stage_config['events']
+        # Run even events are not defined to remove previously existing ones (thus default to []).
+        events = self.stage_config.get('events', [])
 
-            if not isinstance(events, list): # pragma: no cover
-                print("Events must be supplied as a list.")
-                return
+        if not isinstance(events, list): # pragma: no cover
+            print("Events must be supplied as a list.")
+            return
 
-            try:
-                function_response = self.zappa.lambda_client.get_function(FunctionName=self.lambda_name)
-            except botocore.exceptions.ClientError as e: # pragma: no cover
-                print("Function does not exist, please deploy first. Ex: zappa deploy {}".format(self.api_stage))
-                return
+        if not events:
+            print("No events defined in zappa config. Proceeding to unschedule to remove leftovers (in case "
+                  "configuration was changed).")
```

Comment:
Is there a case where the user needs to see this in the logs / can we just silently continue here?